### PR TITLE
Update: Built API

### PR DESCRIPTION
**Pull request by ramaneswaran**

### CODE REVIEW

File: app/genius/__init__.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            songs.append(song)
+            
+        return songs
+    
+    def search_artist(self, query, max_songs=10):
+        '''
+        Search for artist
+        '''
+        try:
+            query = '%20'.join(query.split(' '))
```
Comment (Geek-ubaid) [inline app/genius/__init__.py]: use urlencode for making query param string

**Review by Geek-ubaid — CHANGES_REQUESTED** [review]

File: app/genius/__init__.py (hunk 1 above)
Comment (ramaneswaran) [reply app/genius/__init__.py]: All right will do, tbh i didn't know urlencode existed

**Comment by Geek-ubaid** [comment]
@ramaneswaran Update the PR with the requested changes its getting stale

**Comment by ramaneswaran** [comment]
Added urlencode

**Review by Geek-ubaid — APPROVED** [review]

**Pull request merged**